arbiter-core/src/middleware/connections.rs

```diff
@@ -1,4 +1,4 @@
-///! Messengers/connections to the underlying EVM in the environment.
+/// ! Messengers/connections to the underlying EVM in the environment.
```

Comment:
I'm confused as to all the `/// !` comments. 

Module level doc comments are `//!` and other doc comments are usually just `///`

Reply:
Same as above